Kap/Kap.download.recipe

```diff
@@ -10,6 +10,9 @@
 	<dict>
 		<key>NAME</key>
 		<string>Kap</string>
+		<key>ARCH</key>
+		<string></string>
+		<!-- VALID arch FOR THIS RECIPE ARE: x64 (INTEL), arm64 (APPLE SILICON) -->
```

Comment:
Please move this into the description field.

Reply:
Done